groth16/Cargo.toml

```diff
@@ -23,7 +23,7 @@ edition = "2018"
 ################################# Dependencies ################################
 
 [dependencies]
-algebra = { path = "../algebra", features = [ "parallel" ] }
+algebra = { path = "../algebra", features = [ "parallel", "std" ] }
```

Comment:
```suggestion
algebra = { path = "../algebra", features = [ "parallel" ] }
```

Reply:
Though, if you want these crates to also be `no_std`, you probably need to drop the `parallel` feature.

I'd recommend adding a `parallel` feature to this crate, and then making that activate `algebra/parallel`.

Reply:
(same for `gm17`)

Reply:
I'll work on this in the next PR